hedgehog/src/Hedgehog/Internal/Gen.hs

```diff
@@ -256,8 +259,7 @@ runGenT size seed (GenT m) =
 --
 evalGen :: Size -> Seed -> Gen a -> Maybe (Tree a)
 evalGen size seed =
-  fmap (fmap Maybe.fromJust) .
-  Tree.filter Maybe.isJust .
+  Tree.mapMaybe id .
```

Comment:
This is a substantial change. Please be especially careful to check that it makes sense!

Reply:
Provided `Tree.mapMaybe` works then this looks like it would be fine.